src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
```

Comment:
Avoid using tag names for styling, except for `html` and `body`. Consider using a class for the body element instead.